05_toolkit/code/exercise_5a_solution/utils/strings_test.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import "testing"
+
+func TestMakeExcited(t *testing.T) {
+	expected := "OMG SO EXCITING!"
+	actual := MakeExcited("omg so exciting")
+	if actual != expected {
+		t.Errorf("Average was incorrect! Expected: %s, Actual: %s", expected, actual)
```

Comment:
Should probably modify this error line to be something like:

`t.Errorf("String was incorrectly formatted! Expected: %s, Actual: %s", expected, actual)`

Reply:
This is the solution that appears in the course -- just to prevent confusion from students, unless you think it's more confusing to leave it as it is, I would prefer to match the course. Let me know what you think!

Reply:
That makes sense - completely my fault there for the discrepency.